taskcluster/ci/android-build/kind.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         - [source, taskcluster/scripts/toolchain/rustup-setup.sh]
         - [source, taskcluster/scripts/toolchain/cross-compile-setup.sh]
         - [rsync, '-a', /builds/worker/fetches/libs/, /builds/worker/checkouts/src/libs/]
-        - [bash, '-c', 'echo "rust.targets=linux-x86-64,x86_64\n" > local.properties']
+        - [bash, '-c', 'echo "rust.targets=linux-x86-64\n" > local.properties']
```

Comment:
This is the task that gets run on an ordinary PR, and thankfully it was only building the megazord twice rather than 70+ times. Still, I think we can save a bit of time here. We only need to build for the native platform in order to run the tests, so I've removed `x86_64` from the target list here.

You could argue that there's some value in ensuring that the megazord *builds* correctly on the mobile target platforms on every PR, but in that case I think we should have all the target platforms here rather than just this one. Unless it was here for some other reason that I don't anticipate?